# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: toolkit/components/extensions/test/xpcshell/test_ext_webRequest_urlclassification.js
# Commit: f2df73fda468
# Full Hash: f2df73fda46859150605f33bcdaeb49ab7037720
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2019-10-19 21:40:34
# Regressor Bug: 1588899
# File Overlap Count: 1
# Description:
#   Bug 1588899 - P1. Move classification flags related method to nsIClassifiedChannel. r=Ehsan,baku
#   
#   This is where it should have been in the first place. Those attributes belong there.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D49577
# ==============================================================================

diff -r b98968eb71d3 -r f2df73fda468 toolkit/components/extensions/test/xpcshell/test_ext_webRequest_urlclassification.js
--- a/toolkit/components/extensions/test/xpcshell/test_ext_webRequest_urlclassification.js	Thu Oct 17 15:24:31 2019 +0000
+++ b/toolkit/components/extensions/test/xpcshell/test_ext_webRequest_urlclassification.js	Sat Oct 19 04:30:24 2019 +0000
@@ -3,7 +3,7 @@
 const { Schemas } = ChromeUtils.import("resource://gre/modules/Schemas.jsm");
 
 /**
- * If this test fails, likely nsIHttpChannel has added or changed a
+ * If this test fails, likely nsIClassifiedChannel has added or changed a
  * CLASSIFIED_* flag.  Those changes must be in sync with
  * ChannelWrapper.webidl/cpp and the web_request.json schema file.
  */
@@ -20,7 +20,7 @@
 
   let prefix = /^(?:CLASSIFIED_)/;
   let entries = 0;
-  for (let c of Object.keys(Ci.nsIHttpChannel).filter(name =>
+  for (let c of Object.keys(Ci.nsIClassifiedChannel).filter(name =>
     prefix.test(name)
   )) {
     let entry = c.replace(prefix, "").toLowerCase();